cou/commands.py

```diff
@@ -151,7 +151,7 @@ def get_dataplane_common_opts_parser() -> argparse.ArgumentParser:
         action="append",
         help="Specify machine id(s) to upgrade.\nThis option accepts a single machine id as well "
         "as a stringified comma-separated list of ids,\nand can be repeated multiple times.",
-        dest="machines",
+        dest="raw_machines",
```

Comment:
Instead of this, we can define our own action to get all machines we need.

```python
class SplitArgs(argparse._AppendAction):
    def __call__(self, parser, namespace, values, option_string=None):
        for value in values.split(","):
            super().__call__(parser, namespace, value, option_string)
...
    dp_upgrade_group.add_argument(
        "--machine",
        "-m",
        action=SplitArgs,
...
```
Like this, we can parse machine defined as list separated by comma or multiple arguments.


However, I do not see a reason why we are supporting both? It would not be enough to support only one way? e.g. `--machine 1 --machine 2` or `--machines 1,2`